auto-self-nominate/src/AccountState.ts

```diff
@@ -0,0 +1,100 @@
+import { SDK } from "codechain-sdk";
+import { PlatformAddress } from "codechain-sdk/lib/core/classes";
+
+const RLP = require("rlp");
+
+export type AccountState = "Banned" | "Candidates" | "Jailed";
+export interface JailedAccountInfo {
```

Comment:
Why not using type alias?
```ts
export type JailedAccountInfo = {
```

Reply:
OMG. Formatter changes my type alias to interface. I changed tslint option.